### PR TITLE
Utils Unit Tests

### PR DESCRIPTION
**Pull request by spmcbride1201**
Adding new unit tests for getRouteMetrics and getRouteTree Reselect utility functions.

### CODE REVIEW

**Comment by spmcbride1201** [comment]
Slammed through all resistance and got CircleCI 2.0 working. Test Coverage report is at https://251-85883218-gh.circle-artifacts.com/0/home/circleci/repo/coverage/lcov-report/index.html. w00t

File: src/json/mockReduxState.js [hunk 1]
```diff
@@ -0,0 +1,1682 @@
+export default {
```
Comment (jmcudd) [inline src/json/mockReduxState.js]: Is this mock data based off snapshot of a real state while the app is running?

File: src/utils.test.js [hunk 2]
```diff
@@ -116,3 +35,103 @@ describe("Reselect selector getThreadCounts", () =>
       stopped: 3
     });
   }));
+
+describe("Reselect selector getRouteMetrics", () =>
```
Comment (jmcudd) [inline src/utils.test.js]: Makes sense, you pass in the mock state into `getRouteMetrics` and test for valid outputs.

File: src/utils.test.js [hunk 3]
```diff
@@ -116,3 +35,103 @@ describe("Reselect selector getThreadCounts", () =>
...
+        "1500416074215": 0,
+        "1500416089215": 0,
+        "1500416104216": 0,
+        "1500416119215": 0,
+        "1500416134216": 0
+      }
+    });
+  }));
+
+describe("Reselect selector getRouteTree", () =>
```
Comment (jmcudd) [inline src/utils.test.js]: Straight forward.

File: src/json/mockReduxState.js (hunk 1 above)
Comment (spmcbride1201) [inline src/json/mockReduxState.js]: Yeah, I basically just took a snapshot of the data with everything working as expected.

**Pull request merged**